src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/action/CloseActionIT.kt

```diff
@@ -68,4 +68,46 @@ class CloseActionIT : IndexStateManagementRestTestCase() {
 
         assertEquals("close", getIndexState(indexName))
     }
+
+    fun `test already closed index`() {
+        val indexName = "${testIndexName}_index"
+        val policyID = "${testIndexName}_testPolicyName"
+        val actionConfig = CloseActionConfig(null, null, 0)
+        val states = listOf(
+            State("CloseState", listOf(actionConfig), listOf())
+        )
+
+        val policy = Policy(
+            id = policyID,
+            description = "$testIndexName description",
+            schemaVersion = 1L,
+            lastUpdatedTime = Instant.now().truncatedTo(ChronoUnit.MILLIS),
+            defaultNotification = randomDefaultNotification(),
+            defaultState = states[0].name,
+            states = states
+        )
+        createPolicy(policy, policyID)
+        createIndex(indexName, null)
+        closeIndex(indexName)
+
+        assertEquals("close", getIndexState(indexName))
+
+        addPolicyToIndex(indexName, policyID)
```

Comment:
NP: Should add policy before closing the index initially since this creates an unrealistic workflow. Normally if someone's tries to add a policy to a closed index via the API, it will fail, so it might be confusing for that to pass here.